core/src/sigverify.rs

```diff
@@ -31,18 +31,17 @@ impl Default for TransactionSigVerifier {
 
 impl SigVerifier for TransactionSigVerifier {
     fn verify_batch(&self, mut batch: Vec<Packets>) -> Vec<Packets> {
-        let r = sigverify::ed25519_verify(&batch, &self.recycler, &self.recycler_out);
+        let r = sigverify::ed25519_verify(&mut batch, &self.recycler, &self.recycler_out);
         mark_disabled(&mut batch, &r);
```

Comment:
This is redundant

Reply:
It's still necessary for the gpu case

Reply:
Maybe move this inside `sigverify::ed25519_verify` then? Seems wasteful to do the disabled check twice